scripts/run-fork-chain.sh

```diff
@@ -106,16 +107,17 @@ then
         cargo build --release -p moonbeam
         cp target/release/moonbeam $BINARY_PATH
     fi
+fi
 
-    echo "Preparing tests... (3 minutes)"
-    cd $ROOT_FOLDER/moonbeam/moonbeam-types-bundle
-    npm install
-    cd $ROOT_FOLDER/moonbeam/tools
-    npm install
+echo "Preparing tests... (3 minutes)"
+cd $ROOT_FOLDER/moonbeam/moonbeam-types-bundle
+npm install
+cd $ROOT_FOLDER/moonbeam/tools
+npm install
+
+cd $ROOT_FOLDER/moonbeam/tests
+npm ci
```

Comment:
These have been moved out of the if block, because there are times where you want to skip compilation to avoid the node rebuild - but you still do want to run your tests.

Currently if you have skip compilation=true, it won't work unless you've previously finished that compilation on a previous run.